php/pantheon/checks/sessions.php

```diff
@@ -17,7 +17,7 @@ public function init() {
 		$this->has_plugin = class_exists("Pantheon_Sessions");
 		// If the plugin was not found, define the recommended action.
 		// Otherwise, we don't want to recommend anything, we're all good here.
-		$this->action = ! $this->has_plugin ? 'You should install the Native PHP Sessions plugin - https://wordpress.org/plugins/wp-native-php-sessions/' : 'No action required';
+		$this->action = ! $this->has_plugin ? 'You should INSTALL and ACTIVATE the Native PHP Sessions plugin - https://wordpress.org/plugins/wp-native-php-sessions/' : 'No action required';
```

Comment:
I don't know that we need to capslock `INSTALL`/`ACTIVATE`. 
```suggestion
		$this->action = ! $this->has_plugin ? 'You should ensure that the Native PHP Sessions plugin is installed and activated - https://wordpress.org/plugins/wp-native-php-sessions/' : 'No action required';
```